sydent/db/profiles.py

```diff
@@ -38,3 +38,22 @@ def addBatch(self, host, batchnum, batch):
             vals,
         )
         self.sydent.db.commit()
+
+    def getProfilesMatchingSearchTerm(self, search_term, limit):
+        cur = self.sydent.db.cursor()
+
+        sql = (
+            "SELECT user_id, display_name, avatar_url FROM profiles WHERE "
+            "user_id LIKE LOWER(?) OR "
```

Comment:
don't we need lower(user_id) to handle the `@Amandine:matrix.org` mess?

Reply:
actually, sqlite's LIKEs are case insensitive for ASCII characters by default anyway, i think?  And then collation-aware case sensitivity becomes a complete trainwreck.  It kinda feels like we should be switching to postgres if this is going to have a hope in hell of working, especially for users with accents in their display-names...

```
sqlite> select 'a' like 'a';
1
sqlite> select 'A' like 'a';
1
sqlite> select 'Á' like 'a';
0
sqlite> select 'Á' like 'á';
0
sqlite> select 'Á' like 'Á';
1
```

That said, i don't think this should block shipping an initial cut.

Reply:
also:

```
sqlite> select lower('Á');
Á
```

:/